File: Semana1/variaveis/variaveis_java/Objeto.java
package variaveis_java;

class Objeto{
    static String variavelClasse = "Essa eh uma variavel de classe";
    /*
    Variáveis de classe são compartilhadas por todas as instâncias, por isso são precedidas
    pela palavra reservada static
    */
    String variavelObjeto = "Essa eh uma variavel de objeto";



    public String getVarClasse(){
        return variavelClasse;
    }

    public String getVarObjeto(){
        return variavelObjeto;
    }
}

File: Semana1/variaveis/variaveis_python/Objeto.py
class Objeto:
    variavelClasse = "Essa eh um variavel de classe"

    #CONSTRUTOR
    def __init__(self, variavel):
        # self indica o objeto em particular
        self.variavelObjeto = variavel
        #Objeto. indica a variavel de classe
        Objeto.variavelClasse = variavel

    def setVariavelClasse2(self, variavel):
        self.variavelClasse = variavel

    def imprime(self):
        print('Essa eh a variavei de objeto:', self.variavelObjeto, end="\n")
        print('Essa eh a variavei de classe:', self.variavelClasse, end="\n")

        



